### PR TITLE
CodeWhisperer: factor in language dialect for crossfile context fetching

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavaCodeWhispererFileCrawler.kt [hunk 1]
```diff
@@ -21,6 +21,7 @@ import org.jetbrains.jps.model.java.JavaModuleSourceRootTypes
 object JavaCodeWhispererFileCrawler : CodeWhispererFileCrawler() {
     override val fileExtension: String = "java"
     override val testFilenamePattern: Regex = """(?:Test([^/\\]+)\.java|([^/\\]+)Test\.java)$""".toRegex()
+    override val dialects: Set<String> = setOf(".java")
```
Comment (andrewyuq) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavaCodeWhispererFileCrawler.kt:24]: this one has a dot `.` while the others don't

Comment (Will-ShaoHua) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavaCodeWhispererFileCrawler.kt:24]: sorry, my miss.

**Review by vchikoti1998 — APPROVED** [review]
Does this not need a change log?

**Comment by Will-ShaoHua** [comment]
> Does this not need a change log?

Nope Vineeth, we don't this one for this

**Review by andrewyuq — APPROVED** [review]

**Pull request merged**